google/earthenterprise
# 1515 fix always skipped test

Removes what an old unused sample test. This test was always skipped in geecheck. Now it will no longer incorrectly report that fusion is not installed.

To test:

1. Build and install OpenGEE.
2. Navigate to the geecheck page.
3. Verify that testWithSampleCondition no longer appears.
